kubefirst/manifest.yaml

```diff
@@ -2,6 +2,6 @@
 name: kubefirst
 version: "2.1.19"
 maintainer: "@kubefirst"
-description: kubefirst is an instant kubernetes platform for engineering teams
+description: kubefirst is an instant open source Kubernetes platform powered by the most popular Cloud Native tools to provide app delivery and automated infrastructure as code
```

Comment:
```suggestion
description: kubefirst is an instant open source Kubernetes platform powered by the most popular, useful, and mission critical CNCF/Cloud Native tools to provide app delivery and automated infrastructure as code
```

Reply:
i agree with the addition except the space it has to fit into. need to discard the solid recommendation clay, thanks for the proposal though